scenarios/reproducers/va-pidone.yml

```diff
@@ -139,3 +143,18 @@ cifmw_test_operator_tolerations:
     effect: "NoExecute"
 cifmw_test_operator_node_selector:
   kubernetes.io/hostname: worker-3
+
+# Use EDPM image for computes
+cifmw_update_containers_edpm_image_url: "{{ cifmw_update_containers_registry }}/{{ cifmw_update_containers_org }}/edpm-hardened-uefi:{{ cifmw_update_containers_tag }}"
```

Comment:
Are you sure you need this one that is already declared in the release files in DS?

Reply:
I just thought it would be nice to have it in case we run outside of zuul (and using upstream content).